TDL-06-EC2-Cloudwatch-S3/terraform-code-ec2-cw-s3/security-group.tf

```diff
@@ -0,0 +1,24 @@
+resource "aws_security_group" "instance_sg" {
+  name        = "${var.project_name}-instance-sg"
+  description = "Allow http traffic and all egress traffic"
+  vpc_id      = var.vpc_id
+
+  ingress {
+    description = "Inbound HTTP"
+    from_port   = 80
+    to_port     = 80
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
```

Comment:
By default, SG allows all egress traffic, so we don't need egress here if we allow all outbound traffic.